os_net_config/impl_nmstate.py

```diff
@@ -1011,9 +1011,11 @@ def add_interface(self, interface):
 
         logger.info(f'adding interface: {interface.name}')
         data = self._add_common(interface)
-        if isinstance(interface, objects.Interface):
-            data[Interface.TYPE] = InterfaceType.ETHERNET
-            data[Ethernet.CONFIG_SUBTREE] = {}
+
+        data[Interface.TYPE] = InterfaceType.ETHERNET
+        data[Ethernet.CONFIG_SUBTREE] = {}
+        data[Ethernet.CONFIG_SUBTREE][Ethernet.SRIOV_SUBTREE] = {
+            Ethernet.SRIOV.TOTAL_VFS: 0}
```

Comment:
Should we add  Ethernet.SRIOV.VFS_SUBTREE: [], like:
        data[Ethernet.CONFIG_SUBTREE][Ethernet.SRIOV_SUBTREE] = {
            Ethernet.SRIOV.TOTAL_VFS: 0,
            Ethernet.SRIOV.VFS_SUBTREE: [],
         }

Reply:
When total_vfs is 0, it also means there are no vfs. isn't it ?